width: 848
height: 800
frame_rate: 30

initial_exposure: 5000
initial_gain: 16

max_exposure: 12000
mean_intensity_setpoint: 1800
laser_power: 0

exposure_change_rate: 1.5
target_mean: 110
dead_region: 5

a_min: 0.4
a_max: 0.9
c: 0.5